jinahub/encoders/text/TransformerTorchEncoder/transform_encoder.py

```diff
@@ -88,6 +99,14 @@ def __init__(
         )
         self.model.to(torch.device(device))
 
+        self.is_distill_bert = self.model.config.model_type == 'distilbert'
```

Comment:
PRs should be, if possible, limited to a single issue. The title of your PR is "support multi-gpus" - I think fixing the encoder to support distilbert should be a separate PR

Reply:
thanks for your suggestion. Anyway, i found this is a false alarm (the related issue #16 has been closed). I have reverted this change.

Reply:
All good then!